catalyst_voices/apps/voices/lib/widgets/cards/proposal_card_widgets.dart

```diff
@@ -35,17 +37,70 @@ class FinalProposalChip extends StatelessWidget {
           color: Theme.of(context).colorScheme.onPrimary,
         ),
       ),
-      backgroundColor: Theme.of(context).colorScheme.primary,
+      backgroundColor:
+          onColorBackground ? Theme.of(context).colorScheme.primary : null,
+    );
+  }
+}
+
+class PrivateProposalChip extends StatelessWidget {
+  const PrivateProposalChip({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return VoicesChip.rectangular(
+      content: Text(
+        context.l10n.private,
+        key: const Key('ProposalStage'),
+        style: TextStyle(
+          color: Theme.of(context).colors.textOnPrimaryLevel1,
+        ),
+      ),
+      backgroundColor:
+          Theme.of(context).colors.elevationsOnSurfaceNeutralLv1Grey,
+    );
+  }
+}
+
+class ProposalCommentsChip extends StatelessWidget {
+  final int commentsCount;
+  final bool useInternalBackground;
+  const ProposalCommentsChip({
+    super.key,
+    required this.commentsCount,
+    this.useInternalBackground = true,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return VoicesChip.rectangular(
+      backgroundColor: useInternalBackground
+          ? context.colors.elevationsOnSurfaceNeutralLv1Grey
+          : null,
+      leading: VoicesAssets.icons.chatAlt2.buildIcon(
+        color: useInternalBackground ? null : context.colors.iconsBackground,
+      ),
+      content: Text(
+        key: const Key('CommentsCount'),
+        commentsCount.toString(),
+        style: context.textTheme.labelLarge?.copyWith(
+          color: useInternalBackground
+              ? Theme.of(context).colors.textOnPrimaryLevel1
+              : context.colors.iconsBackground,
+        ),
+      ),
     );
   }
 }
 
 class ProposalVersionChip extends StatelessWidget {
+  final bool useInternalBackground;
   final String version;
```

Comment:
I'd keep the same hierarchy as in constructor, especially since this bool param is optional
```suggestion
  final String version;
  final bool useInternalBackground;
```